www/install/php/Update-21.10.4.php

```diff
@@ -19,11 +19,52 @@
  *
  */
 
+
 include_once __DIR__ . "/../../class/centreonLog.class.php";
 $centreonLog = new CentreonLog();
 
+//error specific content
 $versionOfTheUpgrade = 'UPGRADE - 21.10.4: ';
 
+/**
+ * Query with transaction
+ */
+try {
+    $pearDB->beginTransaction();
+    $errorMessage = 'Unable to delete logger entry in cb_tag';
+    $statement = $pearDB->query("DELETE FROM cb_tag WHERE tagname = 'logger'");
+    $errorMessage = 'Unable to update the description in cb_field';
+    $statement = $pearDB->query("
+        UPDATE cb_field
+        SET `description` = 'Time in seconds to wait between each connection attempt. The default value is 30s.'
```

Comment:
Time in seconds to wait between each connection attempt (Default value: 30s).
```suggestion
        SET `description` = 'Time in seconds to wait between each connection attempt (Default value: 30s).'
```